src/TestSuite/Fixture/FixtureManager.php

```diff
@@ -94,12 +94,9 @@ public function setDebug($debug)
     public function fixturize($test)
     {
         $this->_initDb();
-        if (empty($test->fixtures) || !empty($this->_processed[get_class($test)])) {
+        if (empty($test->getFixtures()) || !empty($this->_processed[get_class($test)])) {
```

Comment:
This method call should be wrapped by empty().

Reply:
What do you mean? It is already wrapped.

Reply:
should **not**
sry
In this case just use if () or if (!)

Reply:
What's the point of that? `$test->getFixtures()` does not return a boolean. It may return an empty array.

Reply:
http://sandbox.onlinephpfunctions.com/code/5106681ba751da5d3e3edf2429cd754f1c913078

Reply:
The point is that we don't do that, and we shouldn't start now.

Reply:
I don't understand you. `empty($test->fixtures)` is replaced with `empty($test->getFixtures())`. So `empty()` has already been used here. Besides that `!$test->getFixtures()` does not work here as it evaluates to `false` when the fixtures are an empty array. Please take a look at the example in the sandbox I posted above.

Reply:
empty() on a method name is the issue. Before it was not a method. And `!` usage is the non cloakling equal to that. 100% same result. Please do not argue here on this. This has always been this way.